src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,329 @@
+* [ImageCapture implementation status](https://github.com/w3c/mediacapture-image/blob/gh-pages/implementation-status.md)
+* [Shape Detection API](https://wicg.github.io/shape-detection-api/#introduction)
+* [Shape Detection explainer and readme](https://github.com/WICG/shape-detection-api)
+* [Face detection demo](https://codepen.io/miguelao/pen/VKOPdX)
+* [Barcode detection demo](https://codepen.io/miguelao/pen/bBWOzM)
+
+{% include "comment-widget.html" %}
+
+<script>
+'use strict';
+
+/* globals ImageCapture */
+
+var constraints;
+var imageCapture;
+
+var grabFrameButton = document.querySelector('#ic-demo button#grabFrame');
+var takePhotoButton = document.querySelector('#ic-demo button#takePhoto');
+
+var canvas = document.querySelector('#ic-demo canvas');
+var img = document.querySelector('#ic-demo img');
+var video = document.querySelector('#ic-demo video');
+var videoSelect = document.querySelector('#ic-demo select#videoSource');
+var zoomInput = document.querySelector('#ic-demo input#zoom');
+
+grabFrameButton.onclick = grabFrame;
+takePhotoButton.onclick = takePhoto;
+videoSelect.onchange = getStream;
+zoomInput.oninput = setZoom;
+
+// Get a list of available media input (and output) devices.
+navigator.mediaDevices.enumerateDevices().then(gotDevices).
+catch(error => {
+  console.log('Error getting devices: ', error);
+});
+
+// Get a video stream from the currently selected camera source.
+function getStream() {
+  if (window.stream) {
+    window.stream.getTracks().forEach(track => {
+      track.stop();
+    });
+  }
+  var videoSource = videoSelect.value;
+  constraints = {
+    audio: false,
+    video: {deviceId: videoSource ? {exact: videoSource} : undefined}
+  };
+  navigator.mediaDevices.getUserMedia(constraints).
+  then(gotStream).then(gotDevices).catch(error => {
+    console.log('getUserMedia error: ', error);
+  });
+}
+
+// From the list of media devices available, set up the camera source <select>,
+// then get a video stream from the default camera source.
+function gotDevices(deviceInfos) {
+  for (var i = 0; i !== deviceInfos.length; ++i) {
+    var deviceInfo = deviceInfos[i];
+    console.log('Found media input or output device: ', deviceInfo);
+    var option = document.createElement('option');
+    option.value = deviceInfo.deviceId;
+    if (deviceInfo.kind === 'videoinput') {
+      option.text = deviceInfo.label || 'Camera ' + (videoSelect.length + 1);
+      videoSelect.appendChild(option);
+    }
+  }
+  getStream();
+}
+
+// Display the stream from the currently selected camera source, and then
+// create an ImageCapture object, using the video from the stream.
+function gotStream(stream) {
+  console.log('getUserMedia() got stream: ', stream);
+window.stream = stream; // global scope visible in browser console
+if (window.URL) {
+  video.src = window.URL.createObjectURL(stream);
+  video.classList.remove('hidden');
+} else {
+  video.src = stream;
```

Comment:
`srcObject` should be used to refer to a media provider object (`stream` in this case), whereas `src` should be used with a URL, so:
`video.src = window.URL.createObjectURL(stream);`
or
`video.srcObject = stream;`
should be correct and equivalent.

https://www.w3.org/TR/2015/WD-html51-20150506/semantics.html#dom-media-srcobject